vectordraw/public/js/vectordraw.js

```diff
@@ -419,6 +419,50 @@ function VectorDrawXBlock(runtime, element, init_args) {
         return targetObjects
     }
 
+    VectorDraw.prototype.preventDefault = function(e) {
+        // for disabling scroll http://stackoverflow.com/a/4770179/2747370
```

Comment:
Could you please make the comments for these 4 functions describe what each of them actually do?